e2e/claims.service.e2e.ts

```diff
@@ -220,6 +220,82 @@ describe('Enrollment claim tests', () => {
       return issuedClaim;
     }
 
+    async function enrolAndIssueWithoutRequest(
+      requestSigner: Required<ethers.Signer>,
```

Comment:
I think it is a bit confusing to use `requestSigner` and `requesterDID` in `enrolAndIssueWithoutRequest`. Publishing would be better to be part of the test instead of this method